test/KeyInfo-en-US-windows.json

```diff
@@ -1301,6 +1301,18 @@
         "ConsoleKey":  "UpArrow",
         "Modifiers":  "Shift"
     },
+    {
```

Comment:
The other keyboard layouts will need similar entries.

Reply:
Updated. I installed the keyboard layouts and collect the date by `[Console]::ReadKey()`.